content/03.methods.md

```diff
@@ -106,10 +106,25 @@ Per recommendations by the New York Genome Center [@doi:10.1101/623702], the Lan
 
 We filtered outputs from both callers on the "PASS" filter, and annotated using The ENSEMBL Variant Effect Predictor [@doi:10/gdz75c], reference release 93, and created MAFs using MSKCC's vcf2maf [@url:https://github.com/mskcc/vcf2maf] v1.6.17.
 
+#### Consensus SNV Calling
+
+Our SNV calling process led to separate sets of predicted mutations for each caller.
+We considered mutations to describe the same change if they were identical for the following MAF fields: `Chromosome`, `Start_Position`, `Reference_Allele`,  `Allele`, and `Tumor_Sample_Barcode`.
+We examined the variant allele frequencies produced by each caller and compared their overlap with each other [@url:https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/analyses/snv-callers/compare_snv_callers_plots.Rmd].
+VarDict called many variants that were not called by other callers [@url:https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/analyses/snv-callers/plots/comparison/upset_plot.png], while the other callers produced results that were relatively consistent.
+Numerous VarDict calls were variants with low allele frequency [@url:https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/analyses/snv-callers/plots/comparison/vaf_violin_plot.png].
+We termed mutations shared among the other three callers (Strelka2, Mutect2, and Lancet) to be consensus mutation calls and dropped VarDict due to concerns about its call accuracy in this setting.
```

Comment:
Do we want to say accuracy here, or specifically that it was the potentially large number of false positives that we were concerned about?

Reply:
I've changed this sentence to say false positives instead. I think that's more specifically what we are wanting to refer to here.